circle.yml

```diff
@@ -104,6 +105,10 @@ test:
             > $CIRCLE_ARTIFACTS/server_file_rawnode.txt
         & bash wait_for_local_port.bash 8000 40
         && npm run ft_node
+    - S3BACKEND=file S3VAULT=mem npm start
```

Comment:
I don't think there's much value in monitoring a system that forgets data when it's restarted :)